en/02_Developer_Guides/08_Performance/06_ORM.md

```diff
@@ -49,3 +49,15 @@ SilverStripe\ORM\Connect\DBSchemaManager:
 ```
 
 You can always manually trigger a check and repair (e.g. in a [`BuildTask`](api:SilverStripe/Dev/BuildTask)) by calling [`DB::check_and_repair_table()`](api:SilverStripe\ORM\DB::check_and_repair_table()). This ignores the above configuration.
+
+## Changing `ClassName` column from enum to varchar {#classname-varchar}
+
+On websites with very large database tables it can take a long time to run `dev/build`, which can be a problem when deploying changes to production. This is because the `ClassName` column is an `Enum` type which requires an a `ALTER TABLE` query to be run affecting every row whenever there is a new valid value for the column.
```

Comment:
You used `enum` lowercase in the changelog but `Enum` here. Please pick one. Same with `varchar` vs `Varchar` in the next paragraph.

Reply:
Done, have gone with lowercase